app.rb

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env ruby
+
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+class App
+  def initialize
+    @books = []
+    @person = []
+    @rentals = []
+  end
+
+  def select_opt
+    option = gets.chomp.to_i
+    case option
+    when 1 then list_books
+    when 2 then list_people
+    when 3 then create_person
+    when 4 then create_book
+    when 5 then create_rental
+    when 6 then list_rentals
```

Comment:
- [ ] Please kindly add the 7 option that is for exiting the app. when I select option 7 I get a message `Invalid number, please try again!`, kindly fix that 🙃 😉

<img width="642" alt="Screen Shot 2022-11-10 at 18 12 11" src="https://user-images.githubusercontent.com/74501523/201149170-c5a41b15-e705-42ae-859d-3fbcf6e86aaf.png">